docs/compression_algorithms/Pruning.md

```diff
@@ -27,6 +27,32 @@ Where ![L_j](https://latex.codecogs.com/png.latex?L_j) is j-th convolutional lay
 
 Then during pruning filters with smaller ![G(F_i)](https://latex.codecogs.com/png.latex?G%28F_i%29) importance function will be pruned first.
 
+#### Algorithm statistics
+A model compression can be measured by two main metrics: filter pruning rate and FLOPs pruning rate. While 
+filter pruning rate shows the ratio of removed filters to the total number of filters in the model, FLOPs pruning rate 
```

Comment:
`removed filters to the total number of filters in the model` - it's actual if all_weights parameter is True.
For default it ratio of removed filters to the total number of filters in each prunable convolution.